go.mod

```diff
@@ -1,20 +1,57 @@
 module github.com/mattermost/rotator
 
-go 1.15
+go 1.19
 
 require (
-	github.com/aws/aws-sdk-go v1.36.7
+	github.com/aws/aws-sdk-go v1.44.268
```

Comment:
We switched to the V2 library in the provisioner. It may be a good idea to do the same here, but it's a breaking change in the code, so let's merge this as is and we can do a follow-up if we need to. Just wanted to note it so we were all aware.